doc/setup.md

```diff
@@ -51,7 +51,19 @@ brew install geckodriver
 1. Install dependencies
 
 ```
-sudo apt install nginx supervisor postgresql libpq-dev nodejs
+sudo apt install nginx supervisor postgresql libpq-dev nodejs npm
```

Comment:
npm should come with nodejs

Reply:
I believe both Guy and I experienced that apt installing nodejs did not install npm when installing on a fresh Ubuntu 20.04 system. This seems to indicate that npm is only suggested as a dependency to nodejs but will not automatically install it unless you have enabled installing suggested dependencies by default: [https://packages.ubuntu.com/focal/nodejs](url)

Reply:
I think it might be the other way around, actually; installing `npm` installs `node`. No harm in specifying both explicitly here, though.

Reply:
Maybe we should just swap `nodejs` for `npm` to minimize the number of dependencies we specify?